core/src/mindustry/ui/dialogs/SchematicsDialog.java

```diff
@@ -46,17 +47,25 @@ void setup(){
         cont.table(s -> {
             s.left();
             s.image(Icon.zoom);
-            s.field(search, res -> {
+            searchField = s.field(search, res -> {
                 search = res;
                 rebuildPane[0].run();
-            }).growX();
+            }).growX().get();
         }).fillX().padBottom(4);
 
         cont.row();
 
         cont.pane(t -> {
             t.top();
             t.margin(20f);
+
+            t.update(() -> {
+                if(Core.input.keyTap(Binding.chat) && Core.scene.getKeyboardFocus() == searchField && firstSchematic != null){
```

Comment:
Where is `firstSchematic` defined? This doesn't look like it will compile.

Reply:
Whoops. Sorry. Forgot to check.